### PR TITLE
New documentation for FMI Weather and Sensor Platforms

### PR DESCRIPTION
**Pull request by anand-p-r**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/34455
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/fmi.markdown [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
+title: FMI
+description: Instructions on how to integrate FMI within Home Assistant.
+ha_category:
+  - Weather
+  - Sensor
+ha_release: 0.108.3
```
Comment (klaasnicolaas) [inline source/_integrations/fmi.markdown]:
```suggestion
ha_release: 0.109
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
Is there already a PR in the core repo?

File: source/_integrations/fmi.markdown (hunk 1 above)
Comment (anand-p-r) [reply source/_integrations/fmi.markdown]: Working on it. Plan is to get it in by tomorrow.

Comment (anand-p-r) [reply source/_integrations/fmi.markdown]: I changed the PR to draft. Will mark it for review once everything is in place. Sorry for wasting your time!

Comment (anand-p-r) [reply source/_integrations/fmi.markdown]: Core PR link - https://github.com/home-assistant/core/pull/34213

**Comment by anand-p-r** [comment]
This is the updated PR in core - https://github.com/home-assistant/core/pull/34455. Moved all configuration under a single domain key.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**